Gems/PhysX/Code/Include/PhysX/EditorColliderComponentRequestBus.h

```diff
@@ -82,4 +82,15 @@ namespace PhysX
     };
 
     using EditorColliderComponentRequestBus = AZ::EBus<EditorColliderComponentRequests>;
+
+    class EditorColliderValidationRequests : public AZ::ComponentBus
+    {
+    public:
+        // TODO
+        virtual void ValidateRigidBodyMeshGeometryType() = 0;
+    };
+
+    using EditorColliderValidationRequestBus = AZ::EBus<EditorColliderValidationRequests>;
 }
```

Comment:
nit: Could we add a closing namespace while we're here (clang-format should do this automatically 🙂)